example/linked_list/CMakeLists.txt

```
@@ -0,0 +1,24 @@
+#include_directories(${CMAKE_CURRENT_LIST_DIR})
+
+macro(ADD_EXAMPLE_INCLUDE name)
+    add_executable(example_${name} example_${name}.f90)
+    target_include_drectories(example_${name} ${CMAKE_CURRENT_SOURCE_DIR})
```

There is a typo (but it will not solve your issue)
```suggestion
    target_include_directories(example_${name} ${CMAKE_CURRENT_SOURCE_DIR})
```